src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,111 @@
+package seedu.addressbook.ui;
+
+import static seedu.addressbook.common.Messages.MESSAGE_GOODBYE;
+import static seedu.addressbook.common.Messages.MESSAGE_INIT_FAILED;
+import static seedu.addressbook.common.Messages.MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE;
+import static seedu.addressbook.common.Messages.MESSAGE_USING_STORAGE_FILE;
+import static seedu.addressbook.common.Messages.MESSAGE_WELCOME;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.ui.TextUi;;
+public class Formatter {
+	
+	/** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+    /** Format of a comment input line. Comment lines are silently consumed when reading user input. */
+    private static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
+    
+    /** TextUi object created to showToUser the Formatted text for display */
+    private TextUi textui;

```

Comment:
I think it's better to declare the methods in Formatter as static and call them in TextUI class, instead of introducing textUI as a member of formatter.